README.md

```diff
@@ -51,12 +51,6 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225156778-0d8624cb-fc36-499c-bb31-7d01846142f1.png)

**[OPTIONAL]**

- [ ] It would be nice if you fix your show the rentals method because when i used it then it didn't take any id from me but it showed all the rentanls which was not in School-Library project :thinking: